spec/fixtures/files/bulk_upload_descriptive.csv

```diff
@@ -1,2 +1,2 @@
+druid:rb369bd9414,foo:12345,CSt,marcorg,original cataloging agency,organization,marc,790730,creation,iso8601,19900820141050.0,modification,SIRSI,a111,record origin,Converted from MARCXML to MODS version 3.7 using MARC21slim2MODS3-7_SDR_v2-5.xsl (SUL 3.7 version 2.5 20210421; LC Revision 1.140 20200717),name,"Harvey, Robert Gibson",life dates,1951-,primary,person,xx,marccountry,MODS issuance terms,issuance,monographic,publication,1978,MODS resource types,resource type,notated music,marcform,form,print,extent,score (77 leaves) : music ; 28 cm.,und,iso639-2b,statement of responsibility,by Robert Harvey.,thesis,"D.M.A. project - Department of Music, Stanford University.",https://sul-purl-stage.stanford.edu/bb041bm1345,name,Stanford University,name,Department of Music,organization,main title,Projects,part name,D.M.A. Final,lcsh,topic,Concertos (Piano),genre,Scores,primary,"Concerto for piano, strings, brass and percussion!",title,"Concertos, piano, orchestra",name,"Harvey, Robert Gibson",life dates,1951-,name,uniform,
```

Comment:
It looks like `title2.type` changed from `uniform` to a blank string or nil, and `title2.structuredValue2.value` took the former `title2.type`'s value of `uniform`? Or am I misreading this? 👀

Reply:
nope i goofed, though it turns out not to matter for the tests

but i will fix